_posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.3.0"
+authors:
+- kkondaka
+- dlv
+date: 2023-06-06 10:00:00 -0700
+categories:
+  - releases
+meta_keywords: Data Prepper 2.3.0, ingestion, observability, log pipelines
+meta_description: Data Prepper 2.3.0 with enhancements to expressions, event tagging, enhancements to add_entries processor, s3 sink and tail sampling processor
+---
+
+Data Prepper 2.3.0 is now available for [download](https://opensearch.org/downloads.html#data-prepper)!
+This release introduces a number of changes that improve Data Prepper’s ability to process complex expressions with functions, arithmetic operations, and string operations.
+
+## Enhancements to Data Prepper expressions
+
+Data Prepper 2.3 supports using functions in expressions. A list of supported functions is available in the [Expression syntax](https://opensearch.org/docs/latest/data-prepper/pipelines/expression-syntax/) documentation.
+
+Data Prepper 2.3 supports three types of expressions:
```

Comment:
I would shift the formatting of these sections to be h3 level headings. This way, we can add definitions underneath each section heading, and we don't have to format whose definition under an unordered list. I added suggestions that accomplish that.